Test/git-issues/git-issue-854.dfy

```diff
@@ -0,0 +1,12 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+module M {
+    export provides l   
+    lemma l() ensures true {}
+} 
+module N {
+    import M
+    function f():nat { assert M.l(); 3 } 
```

Comment:
Change indent level to 2 spaces

Add space between `:` and `nat`

Reply:
Fixed.